src/startupscriptgenerator/src/dotnetcore/scriptgenerator.go

```diff
@@ -151,6 +151,12 @@ func (gen *DotnetCoreStartupScriptGenerator) GenerateEntrypointScript(scriptBuil
 		fmt.Printf("Setting up Environment Variables for Application Insights is done.. \n")
 	}
 
+	extensibleCommands := common.ParseExtensibleConfigFile(filepath.Join(gen.AppPath, consts.ExtensibleConfigurationFileName))
```

Comment:
What is the reasoning for this to be here, vs earlier in the `GenerateEntrypointScript` function? It seems like since it is here, the user could not use this config file to configure things like dynamically installed languages (line 137), or the PORT (line 115). That _should_ be fine since they could just configure those with application settings, right?

Reply:
This was also discussed during the design meeting, but feel free to let me know if you still have any questions around this topic.